docs/docs/predict-tutorial.md

```diff
@@ -0,0 +1,327 @@
+
+By default, predictions will be saved to `zamba_predictions.csv`. You can save predictions to a custom path using the `--save-path` argument.
+
+```console
+$ cat zamba_predictions.csv
+filepath,aardvark,antelope_duiker,badger,bat,bird,blank,cattle,cheetah,chimpanzee_bonobo,civet_genet,elephant,equid,forest_buffalo,fox,giraffe,gorilla,hare_rabbit,hippopotamus,hog,human,hyena,large_flightless_bird,leopard,lion,mongoose,monkey_prosimian,pangolin,porcupine,reptile,rodent,small_cat,wild_dog_jackal
+example_vids/eleph.MP4,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
+example_vids/leopard.MP4,0.0,0.0,0.0,0.0,2e-05,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0125,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
+example_vids/blank.MP4,0.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
+example_vids/chimp.MP4,0.0,0.0,0.0,0.0,0.0,0.0,1e-05,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1e-05,4e-05,0.00162,0.0,0.0,0.0,0.0,0.0,2e-05,2e-05,0.0,1e-05,0.0,0.0038,4e-05,0.0
+```
+
+## Step-by-step tutorial
+
+### 1. Specify the path to your videos
+
+Save all of your videos in one folder.
+
+* Your videos should all be saved in formats that are suppored by FFMPEG, [which are listed here](https://www.ffmpeg.org/general.html#Supported-File-Formats_002c-Codecs-or-Features).
+* Your video folder must contain only valid video files, since zamba will try to load all of the files in the directory.
+* Your videos must all be in the top level of the video folder - `zamba` does not extract videos from nested directories.
+
+Add the path to your video folder with `--data-dir`. For example, if your videos are in a folder called `example_vids`:
+
+```console
+$ zamba predict --data-dir example_vids/
+```
+
+In the python package, the data directory is specified when `PredictConfig` is instantiated:
+
+```python
+predict_config = PredictConfig(data_directory='example_vids/')
+```
+
+### 2. Choose a model for prediction
+
+If your camera videos contain species common to central or west Africa:
+
+* Use the [`time_distributed` model](models.md#time-distributed) if your priority is species classification
+* Use the [`slowfast` model](models.md#slowfast) if your priority is blank vs. non-blank video detection
+
+If your videos contain species common to Europe, use the [`european` model](models.md#european).
+
+Add the model name to your command with `--model`. The `time_distributed` model will be used if no model is specified. For example, if you want to use the `slowfast` model to classify the videos in `example_vids`:
+
+```console
+$ zamba predict --data-dir example_vids/ --model slowfast
+```
+
+In the Python package, model is specified when `PredictConfig` is instantiated:
+
+```python
+predict_config = PredictConfig(data_directory='example_vids/', model_name='slowfast')
+```
+
+### 3. Choose the output format
```

Comment:
In this section, we should mention the new param `predict_all_zamba_species` which allows the user to output all zamba species rather than only the species in the labels file (defaults to `True`). For instance, if their labels contain just two of the zamba labels, by default we still output logits for all zamba species. If they set this to `False`, we'll only output predictions for the species in their videos.

Reply:
Isn't this just for training? `predict_all_zamba_species` is only an option in `TrainConfig`.

Reply:
yes that's right - ignore this!